hw/dv/tools/questa/questa_initial_setup.sh

```diff
@@ -0,0 +1,10 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+#!/usr/bin/bash
+# Remove leading "# " from the transcript file creation
+# https://support.sw.siemens.com/en-US/knowledge-base/MG579948?pid=sc%3Asr-open&index=content-external&audience=external
+# This provides compatibility for log file error checking with other supported simulators within Opentitan
+vsim -do "set PrefMain(LinePrefix) """
+rm transcript
```

Comment:
IMO this can be sequenced from within `dvsim` as opposed to having users run a script externally. Please try adding this to `questa.hjson`:

```
  // Remove leading "# " from the transcript file creation
  // pre_build_cmds: ["{job_prefix} vsim -do \"set PrefMain(LinePrefix) \"\"\"",
                      "rm transcript"]

  // pre_run_cmds: ["{job_prefix} vsim -do \"set PrefMain(LinePrefix) \"\"\"",
                    "rm transcript"]
```

In corp environments, EDA tools are not directly accessible in the user's workstation so not everyone is able to run `vsim` and other tool commands directly. Thats what the `{job_prefix}` is for.

Reply:
I'd suggest mixing the quote types to avoid needing the backslashes.

Reply:
The issue is that running the set PrefMain command halts any simulations from continuing. So using it in the pre_build_cmds and pre_run_cmds halts the simulation.

I have seen how you used the run_script command in vcs.hjson and found a way to do it through manual tcl commands. This now works with no initial Questa setup needed. Il remove questa_initial_setup and a this to the merge request.